pkg/ingester/ingester_v2.go

```diff
@@ -438,6 +455,8 @@ func (i *Ingester) v2Push(ctx context.Context, req *client.WriteRequest) (*clien
 	}
 	i.TSDBState.appenderCommitDuration.Observe(time.Since(startCommit).Seconds())
 
+	db.setLastUpdate(time.Now())
```

Comment:
We can use `startCommit` instead of `time.Now()`. When called in a hot path, `time.Now()` is a bit slow, so I would avoid if not strictly required (like in this case).

Reply:
All `time.Now` does is reading a specific address in memory, where kernel updates the current time. We have more `time.Now` calls here (hidden in `time.Since`). Should we optimize them all?

Reply:
Ok, time.Now does little bit more, although not much.

Reply:
_Let's resolve this conversation and focus on real value. Sorry for that._